examples/websocket/src/client.rs

```diff
@@ -63,9 +68,22 @@ fn update(msg: Msg, mut model: &mut Model, orders: &mut impl Orders<Msg>) {
         }
         Msg::MessageReceived(message) => {
             log!("Client received a message");
-            model
-                .messages
-                .push(message.json::<shared::ServerMessage>().unwrap().text);
+
+            if message.contains_text() {
+                model
+                    .messages
+                    .push(message.json::<shared::ServerMessage>().unwrap().text);
+            } else {
+                orders.perform_cmd(async move {
+                    let bytes = message.bytes().await;
+                    bytes.map(Msg::BytesReceived).ok()
+                });
```

Comment:
OT a bit:
Do you think that `update` function should be `async` or that we should add blocking function like `orders.perform_cmd_sync` for similar cases? Or is the current Elm-like API enough / better?

Reply:
Async `update` function would be really great, but I assume that it would require quite a lot of development and it's not really a necessity. Using messaging API (orders) is enough - introducing blocking function is not really necessary here in my opinion.